setup.cfg

```
@@ -1,3 +1,38 @@
+[metadata]
+name = asgiref
+version = attr: asgiref.__version__
+url = http://github.com/django/asgiref/
+author = Django Software Foundation
+author_email = foundation@djangoproject.com
+description = ASGI specs, helper code, and adapters
+long_description = file: README.rst
+license = BSD
+classifiers =
+    Development Status :: 5 - Production/Stable
+    Environment :: Web Environment
+    Intended Audience :: Developers
+    License :: OSI Approved :: BSD License
+    Operating System :: OS Independent
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Topic :: Internet :: WWW/HTTP
```

Whilst we're at it, we could add some project URL's?

```suggestion
    Topic :: Internet :: WWW/HTTP
project_urls =
    Documentation = https://asgi.readthedocs.io/en/latest/
    Further Documentation = https://docs.djangoproject.com/en/dev/topics/async/#async-adapter-functions
    Changelog = https://github.com/django/asgiref/blob/master/CHANGELOG.txt
```